oak_functions_launcher/tests/integration_test.rs

```diff
@@ -56,27 +46,33 @@ async fn test_launcher_virtual() {
     ))
     .await;
 
+    let wasm_path = oak_functions_test_utils::build_rust_crate_wasm("key_value_lookup")
```

Comment:
Perhaps "key_value_lookup" and "oak_functions_linux_fd_bin" could be consts in this file?

Reply:
I'll make some more refactorings shortly which include this, stay tuned.